Alignment/APEEstimation/python/MisalignmentScenarios_cff.py

```diff
@@ -1238,3 +1238,30 @@
         ),
     ),
 )
+
+MisalignmentScenario_PhaseI_PseudoAsymptotic = cms.PSet(
```

Comment:
@mmusich do you mean to the documentation of this PR or to `MisalignmentScenarios_cff.py` itself?